### DISCUSSION TITLE
paulmann-500.45 blinks every 400s

### DISCUSSION THREAD

**Discussion by uafcybs** [Koenkk/zigbee2mqtt#11887] [Devices]
Hello
I have an issue with [Paulmann 500.45 led controller](https://www.zigbee2mqtt.io/devices/500.45.html#paulmann-500.45).
Once integrated into the system, it works fine, except that it turns on at full brightness for 0.5 seconds every 400 seconds... 
The problem is not in the controller because:
a) it does not "blink" if not connected to the zigbee network
b) this is the second such controller with which I have this problem 
Will be appreciated for any hint (what interval of 400s could be ?)

**Comment by Elluts** [comment]
Same here, any news to solve this problem?
